Makefile

```diff
@@ -2,7 +2,7 @@ FOURMOLU_EXTENSIONS := -o -XTypeApplications -o -XTemplateHaskell -o -XImportQua
 
 NIX_SOURCES := $(shell find * -not -path 'dist-newstyle/*' -iname '*.nix')
 CABAL_SOURCES := $(shell find * -not -path 'dist-newstyle/*' -iname '*.cabal')
-HASKELL_SOURCES := $(shell find * -not -path 'dist-newstyle/*' -iname '*.hs')
+HASKELL_SOURCES := $(shell find app src test -iname '*.hs')
```

Comment:
Why is this changed?

Reply:
It turned out to be an artifact of initially having some scratch `.hs` files in my working copy, instead of proper code and tests. I'll revert it back.